### PR TITLE
👷 Add release pre/post PR scripts

### PR DESCRIPTION
**Pull request by fiendish**
Part of #118

### CODE REVIEW

File: scripts/update_versions.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+from datetime import datetime, timezone
+from pathlib import Path
+
+OUR_URLS = (
+    "http://fhir.kids-first.io",
+    "https://fhir.kids-first.io",
+)
+
+
+def bump_ini(filePath, new_version):
```
Comment (znatty22) [inline scripts/update_versions.py:19]:
```suggestion
def bump_ig_version(filePath, new_version):
```

File: scripts/update_versions.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            line
+            if not line.startswith("version ")
+            else f"version = {new_version}\n"
+            for line in f
+        ]
+    with filePath.open(mode="w") as f:
+        f.writelines(body)
+
+
+def bump_json(filePath, new_version):
```
Comment (znatty22) [inline scripts/update_versions.py:31]:
```suggestion
def bump_resource_version(filePath, new_version):
```

File: scripts/update_versions.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        return
+    if body.get("url", "").startswith(OUR_URLS):
+        body["version"] = new_version
+        if "date" in body:
+            body["date"] = datetime.now().astimezone(timezone.utc).isoformat()
+        with filePath.open(mode="w") as f:
+            json.dump(body, f, indent=2)
+
+
+if __name__ == "__main__":
```
Comment (znatty22) [inline scripts/update_versions.py:45]: I think this script (or CLI command if it gets moved into `fhirutil`) needs to be provided 1 input parameter, the path to the IG .ini file. From there it should be able to determine the IG site root (the directory the ini is in) and also the directory(ies) where the resources live. That's what happens in the `fhirutil validate` command. See https://github.com/ncpi-fhir/ncpi-fhir-utility/blob/d4e0eb3f2907f3455e5f535446f436f1976f2039/ncpi_fhir_utility/app.py#L72

Comment (fiendish) [reply scripts/update_versions.py]: https://github.com/kids-first/kf-model-fhir/pull/149/commits/76f48f4049e321a895f1a3edd3495a38338396e4

**Comment by fiendish** [comment]
> What if we made the `update_versions.py` script a CLI command in `ncpi-fhir-utility` - `fhirutil prep-release` (or whatever)?

If you want. Will just need to eliminate the default value for https://github.com/kids-first/kf-model-fhir/pull/149/files#diff-ea10052576afa9c5249f02776549ebecR66

File: .github/prepare_assets.sh [hunk 4]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+pip install .
+fhirutil validate --publisher_opts="-tx n/a" --clear_output site_root/ig.ini
```
Comment (znatty22) [inline .github/prepare_assets.sh:4]: I'm assuming this script runs from the top level git repo directory not from `.github` directory?

Comment (fiendish) [reply .github/prepare_assets.sh:4]: I believe so

**Comment by znatty22** [comment]
Ugh sorry let me fix the pytest issue in a separate PR and then you can rebase after that merges.

**Pull request merged**